NanoVNASaver/NanoVNASaver.py

```diff
@@ -475,7 +489,8 @@ def __init__(self):
                                                                       "\n\n\N{COPYRIGHT SIGN} Copyright 2019 Rune B. Broberg\n" +
                                                                       "This program comes with ABSOLUTELY NO WARRANTY\n" +
                                                                       "This program is licensed under the GNU General Public License version 3\n\n" +
-                                                                      "See https://mihtjel.github.io/nanovna-saver/ for further details"))
+                                                                      "See https://mihtjel.github.io/nanovna-saver/ for further details\n\n" +
+                                                                      "AUTHOR:\nRune B. Broberg - 5Q5R\n\nCONTRIBUTORS:\nOhan Smit, Neilkatin, Carl Tremblay - VA2SAJ"))
```

Comment:
This is good, but needs to be ported to the new About window :-)

Reply:
done